src/ex1/script.js

```diff
@@ -0,0 +1,123 @@
+
+function addTask(){
+    const task = document.getElementById('newTask');
+    // check if entered empty task
+    if (task.value === ''){
+        alert("You forgot to type the task!");
+        return;
+    }
+    if (document.getElementById(task.value) !== null){
+        alert("You already have that task!");
+        task.value = '';
+        return;
+    }
+    if (task.value){
+        // create div for task + delete button
+        const task_section = document.createElement("div");
```

Comment:
I suggest you move the task element creation to a separate function, for readability and reusability